website/components.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @typedef {{
+ *  id?: string,
+ *  name: string,
+ *  href: string,
+ *  status?: 'production' | 'development' | 'experiment' | 'design',
+ *  story?: {
+ *    id?: string,
+ *    name?: string,
+ *    path?: string
+ *  },
+ *  designs?: {
+ *    name?: string,
+ *    href?: string,
+ *    type?: string
+ *  }[]
+ * }} ComponentMeta
+ */
+
+/**
+ * @type {ComponentMeta[]}
+ */
+const components = [
+  {
+    name: 'Button',
+    href: '/components/buttons/',
+    story: {
+      path: 'components-buttons*'
```

Comment:
the path I get for each of the stories is prepended with a `/story/components-{foo}` 

I think if we just update the baseUrl in the `StorybookLink` component to include the `/story/` in the URL these links will work